hooks/002-add-locales.chroot

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -x
+
+set -e
+# Download all the base locale languages and keep
+# only the Gtk and Gnome ones.
+
+ORIG_BASE_DIR="/usr/share/locale-langpack"
+DEST_BASE_DIR="/usr/share/locale"
+
+apt install -y language-pack-gnome-*
+
+for PATH in "$ORIG_BASE_DIR"/*; do
+    LANG=$(/bin/basename "$PATH")
+    DESTINATION="$DEST_BASE_DIR"/"$LANG"/LC_MESSAGES/
+    /bin/mkdir -p "$DESTINATION"
+    for FILENAME in coreutils.mo ibus10.mo gtk40.mo gtk30.mo gtk20.mo gnome-shell.mo gnome-control-center-2.0.mo gnome-desktop-3.0.mo gnome-initial-setup.mo gnome-menus-3.0.mo language-selector.mo xdg-desktop-portal-gnome.mo xdg-desktop-portal.mo ibus-chewing.mo; do
```

Comment:
We should probably ensure we have translations for gdm3, gnome-terminal, and nautilus

Reply:
Done. I presumed that those were in their own snap.